packages/ag-charts-community/src/chart/toolbar/toolbar.ts

```diff
@@ -36,106 +45,113 @@ export class Toolbar extends BaseModuleInstance implements ModuleInstance {
         left: new Set(),
     };
 
+    private groupCallers: Record<ToolbarGroup, number> = {
+        annotations: 0,
+        ranges: 0,
+    };
```

Comment:
A count of how many modules have tried to enable each group. If `> 0` and `toolbar.[group].enabled === true` then it will show. This allows toolbars and groups to be toggled by the `toolbar.enabled`, each group enabled flag and the linked module enabled flag.